README.md

```diff
@@ -15,7 +15,7 @@ Take a look at [https://mquery.tailcall.net](https://mquery.tailcall.net) for a
 
 Unfortunately, you won't find any actual malware there. For demo purposes we
 have indexed the sources of this project - for example, you can find all exceptions
```

Comment:
Make sure to re-index the repository every now and then. Otherwise, I can see how users trying to copy a string from the README.md and can't find it in the demo instance since the string was introduced only later.

Reply:
I don't believe in tedious manual work, so I'd like to integrate this with CI. It just has a low priority so never got around to actually doing that.

But :+1: good point.